src/modules/personenkontext/domain/personenkontext.ts

```diff
@@ -128,8 +128,8 @@ export class Personenkontext<WasPersisted extends boolean> {
     public async checkReferences(): Promise<Option<DomainError>> {
         const [personExists, orga, rolle]: [boolean, Option<Organisation<true>>, Option<Rolle<true>>] =
             await Promise.all([
-                this.personRepo.exists(this.personId),
-                this.organisationRepo.findById(this.organisationId),
+                this.personRepository.exists(this.personId),
+                this.organisationRepository.findById(this.organisationId),
                 this.rolleRepo.findById(this.rolleId),
```

Comment:
Why are they renamed?

Reply:
As long as both, Repos and Repositories exists for these entities, I want to be clear about what is used.